src/templates/calender/hooks/useCalender.tsx

```diff
@@ -0,0 +1,93 @@
+import DATE from "@/constants/date.constant";
+import dayjs from "dayjs";
+import React from "react";
+import useModal from "@/hooks/useModal";
+import useUser from "@/hooks/useUser";
+import { toast } from "react-toastify";
+import {
+  useAddCalenderPlanMutation,
+  useDeleteCalenderPlanMutation,
+} from "../services/mutation.service";
+import CalenderPlanAddModal from "../layouts/CalenderPlanAddModal";
+import { ChangeDateType } from "../types";
+import { CalenderPlanAddQuery } from "../interfaces";
+import { getPlanTypeByPlanName } from "../helpers";
+
+const useCalender = () => {
+  const [currentMonth, setCurrentMonth] = React.useState(dayjs().month());
+  const { openModal } = useModal();
+  const { user } = useUser();
+  const { mutate: deleteMutate } = useDeleteCalenderPlanMutation();
+  const { mutate: addPlanMutate } = useAddCalenderPlanMutation();
+
+  const firstDayOfMonth = () => {
```

Comment:
firstDay 보단 paddingDay가 좀 더 좋은 네이밍일 것 같은데 어떻게 생각하시나요?
```suggestion
  const paddingDayOfMonth = () => {
```

Reply:
좋은것같습니다 추가적으로 훅에 종속될 기능은 아닌 것 같아 헬퍼로 빼주었습니다!